Pages/Addlanguage.cs

```diff
@@ -0,0 +1,169 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using OpenQA.Selenium.Chrome;
+using System.Collections.Generic;
+using System.Linq;
+using SpecProj2.Utilities;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Collections.Specialized.BitVector32;
+using SeleniumExtras.WaitHelpers;
+using FluentAssertions.Primitives;
+
+
+namespace SpecProj2.Pages
+{
+    public class Addlanguage : CommonDriver
+    {
+        public void addlanguage(string language, string level)
```

Comment:
you still not have implemented the POM pattern 

see https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html